src/pages/zh/dev-tools/cli/toolbox/add-metadata-to-token.md

```
@@ -0,0 +1,109 @@
+## 全局标志
+
+| 标志 | 描述 |
+|------|-------------|
+| `-c, --config <value>` | 配置文件路径。默认为 `~/.config/mplx/config.json` |
+| `-k, --keypair <value>` | 密钥对文件或 Ledger 路径（例如：`usb://ledger?key=0`） |
+| `-r, --rpc <value>` | 集群的 RPC URL |
+
+## 示例
+
+1. 添加基本元数据：
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)